tests/units/iop/test_iop_parser.py

```diff
@@ -1061,3 +1061,24 @@ def test_copyright_no_statement_logs(shared_datadir, parser):
                 "log_level": "error",
             },
         ]
+
+
+def test_collaborations(shared_datadir, parser):
+    content = (shared_datadir / "all_fields.xml").read_text()
+    article = ET.fromstring(content)
+    parsed_article = parser._publisher_specific_parsing(article)
+    assert parsed_article["collaborations"] == ["(JNE collaboration)"]
```

Comment:
the collaboration name, should it be with parenthesis?

Reply:
it was added like this in the xml, I removed all special chars